src/lib.rs

```diff
@@ -0,0 +1,956 @@
+// Copyright 2014-2015 The Servo Project Developers. See the COPYRIGHT
```

Comment:
I was wondering whether this was the project you mentioned. Remember to add yourself as author.